test/129038_ais_class_a.js

```diff
@@ -31,6 +31,18 @@ describe('129038 Class A Update', function () {
     delta.updates.length.should.equal(1)
     delta.context.should.equal('vessels.urn:mrn:imo:mmsi:230982000')
   })
+
+  it('special maneuver converts', function () {
+    var msg = JSON.parse(
+      '{"timestamp":"2014-08-15-15:00:01.665Z","prio":"4","src":"43","dst":"255","pgn":"129038","description":"AIS Class A Position Report","fields":{"Message ID":"1","Repeat Indicator":"Initial","User ID":"230982000","Longitude":"25.2026083","Latitude":"60.2176150","Position Accuracy":"High","RAIM":"not in use","Time Stamp":"0","COG":"154.0","SOG":"2.26","Communication State":"2286","AIS Transceiver information":"Channel B VDL reception","Heading":"153.0","Rate of Turn":"0.047","Nav Status":"Under way using engine", "Special Maneuver Indicator": "Engaged in special maneuver"}}'
+    )
+    var tree = mapper.toNested(msg)
+    tree.should.have.nested.property('navigation.state')
+    tree.should.have.nested.property('navigation.specialManeuver.value', 'engaged')
+    var delta = mapper.toDelta(msg)
+    delta.updates.length.should.equal(1)
+    delta.context.should.equal('vessels.urn:mrn:imo:mmsi:230982000')
```

Comment:
Is it worth adding a spec issue and a TODO item with link there for adding a test for schema compliance?

Reply:
https://github.com/SignalK/specification/issues/563